permissions/src/iosMain/kotlin/IOSPermissionsHelper.kt

```diff
@@ -0,0 +1,77 @@
+package com.splendo.kaluga.permissions
+
+/*
+
+Copyright 2019 Splendo Consulting B.V. The Netherlands
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+*/
+
+import com.splendo.kaluga.log.debug
+import com.splendo.kaluga.log.error
+import platform.Foundation.NSBundle
+
+class IOSPermissionsHelper {
+
+    enum class AuthorizationStatus {
+        NotDetermined,
+        Restricted,
+        Denied,
+        Authorized
+    }
+
+    companion object {
+
+        private const val TAG = "Permissions"
+        fun checkDeclarationInPList(bundle: NSBundle, vararg declarationName: String): MutableList<String> {
```

Comment:
This function name is confusing. If it's `check` it should return true/false.
But it returns list (it shouldn't be mutable at first) then it should describe results.

Reply:
Improved naming + made result immutable